office-addin/package.json

```diff
@@ -25,7 +25,8 @@
     "start:web": "office-addin-debugging start manifest.xml web",
     "stop": "office-addin-debugging stop manifest.xml",
     "validate": "office-addin-manifest validate manifest.xml",
-    "watch": "webpack --mode development --watch"
+    "watch": "webpack --mode development --watch",
+    "cp": "cpx \"dist/**/*.*\" ../mai-mind-map-se/dist/addin"
```

Comment:
Is it possible output to the server serve directory instead of copy files?

Reply:
Done, prevent cp by directly changing output dir.